tests/cubeReducers.test.js

```diff
@@ -43,7 +44,8 @@ describe('buildCube reducer', () => {
       type: 'FETCH_DATA_SUCCESS',
       data: data,
       country: 'gb',
-      risk: 1
+      risk: 1,
+      graphId: 'gb/1'
     });
     expect(newStore.countryPerformanceOnRiskViews['gb/1'].isFetching).toBeFalsy()
     expect(newStore.countryPerformanceOnRiskViews['gb/1'].isFetched).toBeTruthy()
```

Comment:
Can we also check here if other component are untouched. 
To  be sure that when selecting selector for one graphs does not make changes for other ones